sla_per_user_system_test.py

```diff
@@ -37,7 +38,7 @@ class SlaPerUserTest(LongevityTest):
                                 ' -schema \'replication(strategy=NetworkTopologyStrategy,replication_factor=3)\' ' \
         '-mode cql3 native user={user} password={password} -rate threads={threads} ' \
         'throttle=10000/s -pop seq={pop}'
-    STRESS_READ_CMD = 'cassandra-stress read cl=ALL duration={duration} -mode cql3 native user={user} ' \
+    STRESS_READ_CMD = 'cassandra-stress read cl=ALL duration={duration} -mode connectionsPerHost=16 cql3 native user={user} ' \
```

Comment:
It may have an impact on results of non-tablets test. Did you run such test?

Reply:
good point, will check

Reply:
in 2024.2 without tablets test pass https://argus.scylladb.com/tests/scylla-cluster-tests/bb6dd4c4-f720-4486-8455-4ac8128a6e8e